logos/logos-workernode/tools/bench_qwen3.py

```
@@ -161,27 +155,13 @@ async def warmup(url: str, model: str):
 
 async def main():
-    if len(sys.argv) < 2 or sys.argv[1] not in ("ollama", "vllm"):
-        print("Usage: python bench_qwen3.py [ollama|vllm]")
-        print("Env overrides: OLLAMA_URL, VLLM_URL")
-        sys.exit(1)
-
-    backend = sys.argv[1]
+    url = os.environ.get("VLLM_URL", "http://localhost:8000/v1/chat/completions")
+    model = "/tmp/qwen3-30b-a3b.gguf"
+    print(f"\n{'='*60}")
+    print(f"BENCHMARK: qwen3:30b-a3b on vLLM (GGUF)")
```

Ruff (0.16.3)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)